uberfire-nio2-backport/uberfire-nio2-impls/uberfire-nio2-jgit/src/main/java/org/uberfire/java/nio/fs/jgit/JGitFileSystem.java

```diff
@@ -85,4 +85,6 @@ void setHadCommitOnBatchState(Path path,
     void clearPostponedWatchEvents();
 
     boolean hasPostponedEvents();
+
+    boolean isInUse();
```

Comment:
I know that the other methods in this interface are not commented, but a javadoc explaining the meaning of this flag would be useful. Also what operations make filesystem to be in use and what implications does it have when it's in use (e.g. what actions will be possible/impossible)?

Reply:
i'd rename it to hasBeenInUse()